helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -74,9 +74,17 @@ controller:
       servicePort: 9913
 
   resources:
+    # these are default resource requests when cluster profile is not known/determined or when cluster is larger than small
     requests:
       cpu: 2
       memory: 2.5Gi
+  profile:
+    # extra small (1) cluster profile - 1 worker node only - resource requests are not being set
+    #small (2) cluster profile - 2-3 worker nodes, max CPU < 4
```

Comment:
```suggestion
    # small (2) cluster profile - 2-3 worker nodes, max CPU < 4
```
What about the other sizes?

Reply:
Well spotted, thanks, updated these docs; cover this on multiple places didn't notice it was outdated here. PTAL